athena-hbase/src/main/java/com/amazonaws/athena/connectors/hbase/HbaseMetadataHandler.java

```diff
@@ -197,7 +198,7 @@ public ListTablesResponse doListTables(BlockAllocator blockAllocator, ListTables
         for (int i = 0; i < tables.length; i++) {
             TableName tableName = tables[i];
             tableNames.add(new com.amazonaws.athena.connector.lambda.domain.TableName(request.getSchemaName(),
-                    tableName.getNameAsString().replace(request.getSchemaName() + ":", "")));
+                    tableName.getNameAsString().toLowerCase(Locale.ENGLISH).replace(request.getSchemaName() + ":", "")));
```

Comment:
can we not do this? so metadata can reflect correctly

Reply:
If the tableName has uppercase characters, not lower casing here would cause an error as Athena TableNames don't support upper case characters.

Reply:
Just called, trino will lowercase it, so it is not needed here. Changing